tests/integration/mixer/outboundtrafficpolicy/traffic_registry_only_test.go

```diff
@@ -59,7 +62,15 @@ func TestOutboundTrafficPolicy_RegistryOnly(t *testing.T) {
 		{
 			Name:     "TCP",
 			PortName: "tcp",
-			Scheme:   scheme.TCP,
+			Expected: Expected{
+				Metric:          "",
+				PromQueryFormat: "",
+				ResponseCode:    []string{},
```

Comment:
Why will there be no metric for this case?

Reply:
Added TODOs with links to https://github.com/istio/istio/issues/22735